feature/mpls/sr/otg_tests/srte_egress/egress_node_forwarding/README.md

```diff
@@ -0,0 +1,218 @@
+# SR-1.2: Egress Node Forwarding for MPLS traffic with Explicit Null label
+
+## Summary
+
+This is to test the forwarding functionality of MPLS traffic with the Explicit Null label on an Egress node 
+in SRTE+MPLS enabled network.
+
+The tests validate that the DUT performs the following actions -
+
+ - DUT is an egress node in SRTE+MPLS network.
+ - DUT will receive traffic with MPLS Explicit Null label 0 and label 2 for IPv4 and IPv6 destination respectively.
+ - DUT will pop the MPLS label and perform IPv4 and IPv6 forwarding.
+
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* Create the following connections:
+* DUT has ingress and egress port connected to the ATE.
+  
+```mermaid
+graph LR; 
+A[ATE:Port1] --Ingress--> B[Port1:DUT:Port2];B --Egress--> C[Port2:ATE];
+```
+
+* ATE Port 1 hosted prefixes:
+  
+  * ATE-Port1 IPv4 address = ATE-P1-Address
+  * Additional Source Address = IPV4-SRC1
+  * Additional Source Address = IPV6-SRC1
+
+* ATE Port 2 hosted prefixes:
+  
+  * ATE-Port2 IPv4 address = ATE-P2-Address
+  * Additional destination address = IPV4-DST1
+  * Additional destination address = IPV6-DST1
+
+*  ATE Port 1 generates below flow types:
+ 
+ * Flow type 1:  Ethernet+MPLS+IPv4+Payload
+  * For the Ethernet Header:
+     * Source MAC address: Unicast Ethernet MAC
+     * Destination MAC address: DUT-Port1 mac address
+  * For MPLS header:
+     * MPLS label 0
+     * EXP set to 0
+     * S set to 1
+     * TTL set to 64  
+  * For the IP header:
+     * Source IP and Destination IP will be IPV4-SRC1 and IPV4-DST1 respectively.
+     * Protocol will be TCP and UDP and source port (> 1024) and destination port will be 443.
+
+ * Flow type 2:  Ethernet+MPLS+IPv6+Payload
+  * For the Ethernet Header:
+     * Source MAC address: Unicast Ethernet MAC
+     * Destination MAC address: DUT-Port1 mac address
+  * For MPLS header:
+     * MPLS label 2
+     * EXP set to 0
+     * S set to 1
+     * TTL set to 64  
+  * For the IPv6 header:
+     * Source IP and Destination IP will be IPV6-SRC1 and IPV6-DST1 respectively.
+     * Protocol will be TCP and UDP and source port (> 1024) and destination port will be 443.
+       
+## Procedure
+
+### Configuration
+                              
+*   Configure Segment Routing Global Block (srgb) lower-bound: 400000 upper-bound: 465000)
+*   Enable MPLS forwarding.
+*   DUT will have a static IPv4 and IPv6 route for IPV4-DST1 / IPV6-DST1 towards ATE Port2.
+
+### Test 
+
+Verify that:
```

Comment:
Can you pease also add the verification step for telemetry check here too that is check for in-pkts, out-pkts and ensure no drop?